pymatgen/entries/entry.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+
+"""
+This module implements equivalents of the basic ComputedEntry objects, which
+is the basic entity that can be used to perform many analyses. ComputedEntries
+contain calculated information, typically from VASP or other electronic
+structure codes. For example, ComputedEntries can be used as inputs for phase
+diagram analysis.
+"""
+from pymatgen.core.composition import Composition
+from monty.json import MSONable
+
+
+__author__ = "Shyue Ping Ong, Anubhav Jain"
+__copyright__ = "Copyright 2011, The Materials Project"
+__version__ = "1.1"
+__maintainer__ = "Shyue Ping Ong"
+__email__ = "shyuep@gmail.com"
+__status__ = "Production"
+__date__ = "Apr 30, 2012"
+
+
+class Entry(MSONable):
+    """
+    An lightweight Entry object containing key computed data
+    for many purposes.
```

Comment:
@ayushsgupta let's not say "computed data" b/c we also have an `ExpEntry` class for experimental data. Maybe say something like "...containing various types of data associated with a specific chemical composition'